pkg/template/core/starlark_value.go

```diff
@@ -4,16 +4,19 @@
 package core
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/k14s/starlark-go/starlark"
 	"github.com/k14s/ytt/pkg/orderedmap"
 )
 
 type StarlarkValueToGoValueConversion interface {
-	AsGoValue() interface{}
+	AsGoValue() (interface{}, error)
 }
 
+var _ = []StarlarkValueToGoValueConversion{&StarlarkValue{}}
```

Comment:
it's better written as

var _ StarlarkValueToGoValueConversion = &StarlarkValue{}